driver-legacy/src/test/functional/com/mongodb/MongoClientSessionSpecification.groovy

```diff
@@ -51,7 +51,7 @@ class MongoClientSessionSpecification extends FunctionalSpecification {
         def clientSession = getMongoClient().startSession()
 
         expect:
-        clientSession.getOriginator() == getMongoClient()
+        clientSession.getOriginator() == getMongoClient().getDelegate();
```

Comment:
This is kind of interesting: since MongoClient is now actually delegating to MongoClientImpl, it's the latter that creates the ClientSession, so it's the latter that's now the originator.  I don't think this is a problem, since the only purpose of this field is to assert that a session is only used with the MongoClient that started it, and that property is preserved.